code/modules/mob/living/carbon/human/human.dm

```diff
@@ -5,6 +5,7 @@
 	icon = 'icons/mob/human.dmi'
 	icon_state = "body_m_s"
 	deathgasp_on_death = TRUE
+	var/mob/living/carbon/human/cannamepick = FALSE

```

Comment:
What...are you doing?

Reply:
making sure that they cant namepick twice, or that other species cant nampick

Reply:
Why are you typecasting a variable only for it to be `TRUE` or `FALSE`.